src/onboarding/create-utils.js

```diff
@@ -39,3 +40,27 @@ export function prepareTransactionCreatorFromAbi(abi, toAddress) {
     }
   }
 }
+
+export async function getGasPrice() {
+  const web3 = getWeb3(web3Providers.default)
+  try {
+    const response = await fetch(
+      'https://ethgasstation.info/json/ethgasAPI.json',
+      {
+        method: 'GET',
+        mode: 'cors',
+        cache: 'no-cache',
+      }
+    )
+    const jsonResponse = await response.json()
+    const priceInWei = web3.utils.toWei(
+      jsonResponse.gasPriceRange[40].toString(), // using '40' as a safe price, this should be set via user input
```

Comment:
```suggestion
      jsonResponse.gasPriceRange[40].toString(), // using '40' as a safe price, this chould be set via user input
```

We should also document what '40' means; I assume it's seconds?